geopandas/io/tests/test_file.py

```diff
@@ -275,6 +273,22 @@ def test_append_file(tmpdir, df_nybb, df_null, driver, ext):
     assert_geodataframe_equal(df, expected, check_less_precise=True)
 
 
+@pytest.mark.xfail
+def test_gpkg_empty_crs(tmpdir):
```

Comment:
We could maybe directly parametrize this test for the different drivers?

Reply:
Done.